pyGHDL/libghdl/utils.py

```diff
@@ -455,3 +456,8 @@ def sequential_iter(n) -> Generator[Any, None, None]:
         return
     else:
         raise Exception("Unknown node of kind {}".format(kind_image(k)))
+
+@export
+@BindToLibGHDL("elab__vhdl_insts__elab_top_unit")
+def elab_top_unit(config: int) -> int:
+    return 0    
```

Comment:
Please add a doc-string to this function.  
See this example: https://github.com/ghdl/ghdl/blob/master/pyGHDL/libghdl/files_map.py#L258

```suggestion
    return 0  # pragma: no cover
```